Kajabi/sage-lib
# feat(hero): image start

## Description
Add the "left" image option for the Hero called "Image start". 

Additionally, I optimized images in the `assets/images" directory.


## Screenshots
|  Tech  |  Before  |  After  | After 2 |
|--------|--------|--------|--------|
| React | ![before - react](https://github.com/Kajabi/sage-lib/assets/24237393/616b2dc7-bddb-462b-ae02-1a241f506067) |.![after - react1](https://github.com/Kajabi/sage-lib/assets/24237393/1605cc2e-6311-4477-a1d2-81931dd07651) | ![after - react2](https://github.com/Kajabi/sage-lib/assets/24237393/98ab8ba4-9e18-4454-aa68-c4ca9fb024de) |
| Rails | ![before - rails](https://github.com/Kajabi/sage-lib/assets/24237393/70b6f270-86a4-4cf0-b27b-3e529da1aafe) |.![after - rails](https://github.com/Kajabi/sage-lib/assets/24237393/17476adc-22a2-4834-9cdc-fe89b1ef90d7) | N/A |

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/component/hero?tab=preview
2. Verify the Rails version is looking and working as expected
3. Navigate to the react version by selecting the option for "React Component" to ensure the linking works
4. Verify the react version is looking and working as expected


## Testing in `kajabi-products`
1. (**LOW**) Added an image start prop. Since that is new and other changes are documentation only, this is not a breaking change.

## Related
[DSS-476](https://kajabi.atlassian.net/browse/DSS-476)


[DSS-476]: https://kajabi.atlassian.net/browse/DSS-476?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
